src/main/resources/assets/sass/_fonts.scss

```diff
@@ -0,0 +1,25 @@
+@font-face {
```

Comment:
Can we do the git submodule song and dance to include https://github.com/alphagov/govuk_template to pull this in that way instead or is it too iffy?